packages/test/test-utils/src/containerRuntimeFactories.ts

```diff
@@ -110,6 +117,11 @@ export class ContainerRuntimeFactoryWithDefaultDataStore
 	 */
 	private readonly provideEntryPoint: (runtime: IContainerRuntime) => Promise<FluidObject>;
 
+	/**
+	 * {@inheritDoc ContainerRuntimeFactoryWithDefaultDataStoreProps.minVersionForCollab}
+	 */
+	private readonly minVersionForCollab: MinimumVersionForCollab | undefined;
+
 	public constructor(props: ContainerRuntimeFactoryWithDefaultDataStoreProps) {
```

Comment:
Curious - What's the overlap between this props type and `LoadContainerRuntimeParams`? Could we reuse that one here?

Reply:
Looks like there is some overlap but it's not 1:1. There are several options on `LoadContainerRuntimeParams` that we don't use in `ContainerRuntimeFactoryWithDefaultDataStoreProps` and one prop (`defaultFactory`) on `ContainerRuntimeFactoryWithDefaultDataStoreProps` that isn't on `LoadContainerRuntimeParams`